tests/unit/amo/components/TestSearchContextCard.js

```diff
@@ -0,0 +1,158 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import {
+  SearchContextCardBase,
+  mapStateToProps,
+} from 'amo/components/SearchContextCard';
+import { searchLoad, searchStart } from 'core/actions/search';
+import { dispatchClientMetadata, fakeAddon } from 'tests/unit/amo/helpers';
+import { getFakeI18nInst } from 'tests/unit/helpers';
+
+
+describe('SearchContextCard', () => {
+  function render(props = {}) {
+    let { store } = props;
+
+    if (!store) {
+      store = dispatchClientMetadata().store;
+      store.dispatch(searchStart({ filters: { query: 'test' } }));
+    }
+
+    return shallow(
+      <SearchContextCardBase
+        {...mapStateToProps(store.getState())}
```

Comment:
You will need to call `.first().shallow()` for each function passed to `compose`. I was thinking we could maybe do it in a less hacky way with a pattern like this:

```js
// in the component:
export wrappers = [
  translate(),
  connect(mapStateToProps),
];

export default compose(...wrappers)(MyComponent);

// In the test:

import MyComponent, { wrappers } from 'src/MyComponent.js'

// In render():
const root = unwrap(<MyComponent {...props} />, { wrappers });

// in unwrap():
let root = root.shallow(); // top-level
wrappers.forEach(() => {
  // Unwrap each HOC:
  root = root.first().shallow();
});
```